packages/devtools/src/commands/elementClick.js

```diff
@@ -35,15 +35,17 @@ export default async function elementClick ({ elementId }) {
         /**
          * check if page load has happened due to click
          */
-        page.once('framenavigated', () => {
+        const frameNavigatedHandler = () => {
             clearTimeout(waitForPageLoadTimeout)
```

Comment:
@christian-bromann  should `dialog` event be removed here as well?

Reply:
Ahh, I completely forgot why I had implemented it that way. I added comments to better explain how this is suppose to work.